platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/AddressBookInitializerTest.java

```diff
@@ -152,7 +154,7 @@ void noStateLoadedFromDiskGenesisStateChangedAddressBook() throws IOException {
         clearTestDirectory();
         final AddressBook configAddressBook = getRandomAddressBook();
         // initial state has currentAddressBook set to configAddressBook
```

Comment:
is this comment still relevant?

Reply:
I swapped out which unit test is setting the addressbook to null in the genesis state.  The name of the unit test sounds like it implies having an address book in the genesis state.